src/settings/environment.inc

```diff
@@ -90,20 +90,20 @@ elseif (isset($_ENV['PLATFORM_APPLICATION'])) {
   }
 }
 elseif (getenv('DOCKER_LOCAL')) {
-    // Local environment settings - see 90-docker-settings.inc.
-    $platform = 'docker';
-    $env = $base_domains[$_SERVER['HTTP_HOST']];
-    $instance = $_SERVER['HTTP_HOST'];
+  // Local environment settings - see 90-docker-settings.inc.
+  $platform = 'docker';
+  $env = $base_domains[$_SERVER['HTTP_HOST']];
+  $instance = $_SERVER['HTTP_HOST'];
 }
 else {
-    // Running under Pipelines
-    $platform = 'pipelines';
-    $env = $base_domains[$_SERVER['HTTP_HOST']];
-    $instance = $_SERVER['HTTP_HOST'];
+  // Running under Pipelines
+  $platform = 'pipelines';
```

Comment:
Shouldn't this be D_PLATFORM_PIPELINES?

Reply:
Same again.
Fixed in the testing + Drush 9 PR  here https://github.com/teamdeeson/d8-quickstart/pull/68

Reply:
Actually, I should have referenced PR https://github.com/teamdeeson/d8-quickstart/pull/69 and not https://github.com/teamdeeson/d8-quickstart/pull/68